### DISCUSSION TITLE
How to customize a voice?

### DISCUSSION THREAD

**Discussion by AIhasArrived** [neonbjb/tortoise-tts#673] [Q&A]
So let's say I don't wanna clone a voice 100% the same,

Can I mix 2 voices and expect to get good results or will it disturb the "trainin process" and degrade the results?
Can I add some "musics" or "sounds" in the training data to "ALTER" the voice I want to create in some ways? Like making it more melodic, more deep, etc etc
Essenstially finding ways to CONTROL and ALTER how a voice will be outputed so you will avoid produced 100% of the voice a person, instead have one custom and unique?
Any thoughts and tips/tricks to know? Thank you!

**Comment by CoderCowMoo** [comment]
It will try to get as close as it can to the training dataset. It currently isn't good at music and stuff because the author specifically filtered that out while trying to get data for the model. You also can mix 2 voices during inference, using the --voices voice1**&**voice2 flag in the CLI.

**Comment by AIhasArrived** [comment]
Hello @CoderCowMoo  I am using RVC right now, would you know how to apply or transfert what you described to there? 
I know bit of code but really have no idea where to apply the command/option you mentioned
